forge-gui/res/cardsfolder/upcoming/atrocious_experiment.txt

```diff
@@ -0,0 +1,8 @@
+Name:Atrocious Experiment
+ManaCost:2 B
+Types:Sorcery
+A:SP$ Mill | NumCards$ 2 | ValidTgts$ Player | TgtPrompt$ Choose a player | SubAbility$ DBDraw | SpellDescription$ Target player mills two cards, draws two cards, and loses 2 life.
```

Comment:
Probably just remove `TgtPrompt$ Choose a player | `

Forge can parse simple stuff like this no problem